src/main/java/calculator/validator/EdgeCharValidator.java

```diff
@@ -0,0 +1,23 @@
+package calculator.validator;
+
+public class EdgeCharValidator {
+    public EdgeCharValidator(String input) {
+        if (input.charAt(0) != '/') validateEdgeChar(input);
+        if (input.charAt(0) == '/') validateEdgeChar(input.substring(5));
+    }
+
+    private void validateEdgeChar(String input) {
+        char firstChar = input.charAt(0);
+        char lastChar = input.charAt(input.length() - 1);
+
+        if (!Character.isDigit(firstChar)) {
+            throw new IllegalArgumentException("첫 문자가 숫자가 아닙니다.");
+        }
+
+        if (!Character.isDigit(lastChar)) {
+            throw new IllegalArgumentException("마지막 문자가 숫자가 아닙니다.");
+        }
+    }
```

Comment:
임시 변수를 만들지 않고 바로 인자로 넣어주면 코드 길이가 많이 줄을 것 같아요!
```suggestion
    private void validateEdgeChar(String input) {
        if (!Character.isDigit(input.charAt(0))) {
            throw new IllegalArgumentException("첫 문자가 숫자가 아닙니다.");
        }
        if (!Character.isDigit(input.charAt(input.length() - 1))) {
            throw new IllegalArgumentException("마지막 문자가 숫자가 아닙니다.");
        }
    }
```